travel-provider/TSP-30-RankTravelOptions/TSP-30-RankTravelOptionsRequest-RequestSchema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "TSP-30-RankTravelOptionsRequest Request Schema",
+  "version": "0.0.1",
+  "description": "Schema for ranking travel options request sent by CWT",
+  "properties": {
+    "returnId": {
+      "type": "integer",
+      "description": "Unique Id for this booking - generated by RMP."
+    },
+    "travelOptions": {
+      "type": "array",
+      "description": "A list of 1 to 9 travel options.",
+      "minItems": 1,
+      "maxItems": 9,
+      "items": {
+        "$ref": "#/definitions/travelOption"
+      }
+    },
+    "providerFeedback": {
+      "type": "object",
+      "description": "If fewer than 9 travel options are provided the reason must be specified.",
+      "properties": {
+        "limitedOptionsReason": {
+          "type": "string"
+        },
+        "limitedOptionsDescription": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "required": [
+    "returnId",
+    "travelOptions"
+  ],
+  "definitions": {
+    "travelOption": {
+      "type": "object",
+      "description": "A travel option that meets the requirements of the return case.",
+      "minItems": 1,
+      "properties": {
+        "flights": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/flight"
+          }
+        },
+        "totalTicketPricePence": {
+          "type": "number"
+        }
+      },
+      "required": [
+        "flights",
+        "totalTicketPricePence"
+      ]
+    },
+    "flight": {
+      "type": "object",
+      "description": "Flight details.",
+      "properties": {
+        "number": {
```

Comment:
Let's rename this to flightNumber. I know the Jira ticket said number but I think flightNumber will be much clearer. I've updated the example in RM-3077